### PR TITLE
Bottom Tab Navigation

### PR DESCRIPTION
**Pull request by carizzuti**
Bottom tab navigation is working. Screens that are not in the bottom tab navigation will still have navigation working. Once the Favorites screen is merged it can be added to the navigator.

### CODE REVIEW

**Review by Dulaya — APPROVED** [review]

**Comment by Dulaya** [comment]
- In BottomTabNavigator.tsx, consider changing the badge: boolean to value?:number like:

  const DisplayTab = (
     iconName: string,
     buttonColor: string,
     value?: number,
  ) => (
      <View>
         <Icon name={iconName} height={size.lg} width={size.lg} fill={buttonColor} />
         {value && <Badge containerStyle={styles.badgeContainer} value={value} />}
      </View>
      );

- Unused import Rating in BottomTabNavigator.tsx

- Everything works find for me. However, this is the first time that I see Tab navigator too so take my opinion for what it's worth.

**Review by Nap1400 — APPROVED** [review]

**Review by aahenry51 — APPROVED** [review]

**Review by raynesmike — APPROVED** [review]

File: mobile/src/components/bottomTabNavigator/BottomTabNavigator.styles.ts [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
-    shadowRadius:20,
-    elevation:20,
+    shadowOpacity: 0.8,
+    shadowRadius: 20,
+    elevation: 20,
   },
-  tabContainer:{
-    padding:6,
+  tabContainer: {
+    padding: 6,
```
Comment (lasharela) [inline mobile/src/components/bottomTabNavigator/BottomTabNavigator.styles.ts:21]: Maybe from size object and closest size?

File: mobile/src/components/bottomTabNavigator/BottomTabNavigator.styles.ts [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
+    paddingBottom: size.sm,
+    shadowColor: color.black,
+    shadowOffset: {
+      width: 0,
+      height: 12,
     },
-    shadowOpacity:0.8,
-    shadowRadius:20,
-    elevation:20,
+    shadowOpacity: 0.8,
```
Comment (lasharela) [inline mobile/src/components/bottomTabNavigator/BottomTabNavigator.styles.ts:16]: once we'll have shadow object will be cool replace this

File: mobile/src/components/bottomTabNavigator/BottomTabNavigator.tsx [hunk 3]
```diff
@@ -1,41 +1,53 @@
...
+  pressFavorites,
+  pressBell,
+  focusTab,
+}) => {
   return (
     <View style={styles.container}>
       <View style={styles.tabContainer}>
-        <TouchableOpacity>
-        <Icon name="Compass" height={iconSize} width={iconSize} fill={fillPin} />
+        <TouchableOpacity onPress={pressHome}>
```
Comment (lasharela) [inline mobile/src/components/bottomTabNavigator/BottomTabNavigator.tsx:22]: For me looks like this 

<View style={styles.tabContainer}>
        <TouchableOpacity onPress={pressCategory}>
          <Icon name="Pin" height={iconSize} width={iconSize} />
        </TouchableOpacity>
      </View>

is repeated. only few props are changing. maybe we'll create a small array with values and use map to iterate?

File: mobile/src/navigation/bottomTabNavigation/BottomTabNavigation.style.ts [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    flexDirection: 'row',
+    backgroundColor: color.white,
+    justifyContent: 'space-around',
+    shadowColor: color.black,
+    shadowOffset: {
+      width: 0,
+      height: 12,
+    },
+    shadowOpacity: 0.8,
+    shadowRadius: 20,
```
Comment (lasharela) [inline mobile/src/navigation/bottomTabNavigation/BottomTabNavigation.style.ts:15]: shadow once again. might be replaced

File: mobile/src/screens/rating/Rating.tsx [hunk 5]
```diff
@@ -1,12 +1,17 @@
...
 
-export const Rating: FC<TRating> = () => {
-    return (
-        <RatingScreenView/>
-    )
-}
\ No newline at end of file
+export const Rating: FC<TRating> = ({navigation}) => {
+  return (
+    //<RatingScreenView/>
```
Comment (lasharela) [inline mobile/src/screens/rating/Rating.tsx:12]: what is this?

**Review by lasharela — APPROVED** [review]

File: mobile/src/screens/rating/Rating.tsx (hunk 5 above)
Comment (carizzuti) [reply mobile/src/screens/rating/Rating.tsx:12]: That was for testing, forgot to change back. 

Fixed now.

File: mobile/src/navigation/bottomTabNavigation/BottomTabNavigation.style.ts (hunk 4 above)
Comment (carizzuti) [reply mobile/src/navigation/bottomTabNavigation/BottomTabNavigation.style.ts:15]: The shadow wasn't even making a difference visually so I removed it completely.

File: mobile/src/components/bottomTabNavigator/BottomTabNavigator.styles.ts (hunk 1 above)
Comment (carizzuti) [reply mobile/src/components/bottomTabNavigator/BottomTabNavigator.styles.ts:21]: That style isn't being used so I deleted it.

**Pull request merged**